docker-compose.yml

```diff
@@ -159,32 +221,23 @@ services:
             ip link set yellow up && \
 
             ip link add br100 type bridge && \
-            ip link set br100 master blue addrgenmode none && \
-            ip link set br100 addr aa:bb:cc:00:00:02 && \
+            ip link set br100 master green addrgenmode none && \
+            ip link set br100 addr aa:bb:cc:00:00:03 && \
             ip link add vni100 type vxlan local 10.0.0.4 dstport 4789 id 100 nolearning && \
             ip link set vni100 master br100 addrgenmode none && \
             ip link set vni100 type bridge_slave neigh_suppress on learning off && \
             ip link set vni100 up && \
             ip link set br100 up && \
 
             ip link add br101 type bridge && \
-            ip link set br101 master green addrgenmode none && \
-            ip link set br101 addr aa:bb:cc:00:00:03 && \
+            ip link set br101 master yellow addrgenmode none && \
```

Comment:
Can we remove br100 and br101 from leaf2?

Reply:
> 

Done. Can you please check now?